install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -179,6 +179,11 @@ There must not be a pre-existing registry in the {product-title} cluster.
 clusters matching the configuration in the inventory hosts file. This is useful
 for cleaning up the {product-title} environment in the case of a failed
 deployment due to configuration errors.
++
+[NOTE]
+====
+The GlusterFS playbooks are not guaranteed to be idempotent. We do not currently support running them more than once for a given installation without deleting the entire GlusterFS installation (including disk data) and starting over. 
```

Comment:
Who is we? Maybe this can be rewritten as:

Running the playbooks more than once for a given installation is currently not supported without . . .

Reply:
Good catch. I will change this.